### PR TITLE
Support choice between different web editors

### PR DESCRIPTION
**Pull request by franzbecker**
Resolves issue #2

### CODE REVIEW

File: generators/app/templates/org.xtext.example.mydsl.web/build.gradle [hunk 1]
```diff
@@ -1,6 +1,14 @@
 plugins {
```
Comment (cdietrich) [inline generators/app/templates/org.xtext.example.mydsl.web/build.gradle]: Looks like you oversaw the dependencies section

Comment (franzbecker) [inline generators/app/templates/org.xtext.example.mydsl.web/build.gradle]: What do you mean?

Comment (cdietrich) [inline generators/app/templates/org.xtext.example.mydsl.web/build.gradle]: The ace dependency

Comment (franzbecker) [inline generators/app/templates/org.xtext.example.mydsl.web/build.gradle]: I don't think that we need a dependency to ace here, do we?!

Comment (cdietrich) [inline generators/app/templates/org.xtext.example.mydsl.web/build.gradle]: There IS a dep to ace , there should be a dependency to cm or non at all ( see dependencies section could not a a comment there from phone )

Comment (franzbecker) [inline generators/app/templates/org.xtext.example.mydsl.web/build.gradle]: Okay got it, yeah you're right I totally oversaw this one.

**Pull request merged**